uk_bin_collection/uk_bin_collection/councils/MorayCouncil.py

```
@@ -1,83 +1,98 @@
-        user_uprn = user_uprn.zfill(8)
-
-        url = f"https://bindayfinder.moray.gov.uk/disp_bins.php?id={user_uprn}"
-
-        # year = datetime.today().year
-        # url = f"https://bindayfinder.moray.gov.uk/cal_{year}_view.php"
-        print(f"Trying URL: {url}")  # Debug
+        user_uprn = str(user_uprn).zfill(8)
+        year = datetime.today().year
 
+        url = f"https://bindayfinder.moray.gov.uk/cal_{year}_view.php?id={user_uprn}"
         response = requests.get(url)
```

Ruff (0.15.9)

[error] Probable use of `requests` call without timeout

(S113)